WeScan/Common/CaptureSession.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CaptureSession.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 23/9/2018.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import Foundation
+
+/// A class containing global variables and settings for this capture session
+final class CaptureSession {
```

Comment:
Using global states like this is usually not a great idea.
Can't we pass the needed states around using dependency injection instead?

Reply:
This was suggested by you and @AvdLee in the initial review. Passing the states around would be complicated, I feel like this is a better solution.

Reply:
Sounds good!